Lab_SeleniumWevDriverBasic.Загрузка

UlyanaRomanova opened the pull request.

UlyanaRomanova committed:
Lab_SeleniumWevDriverBasic.Загрузка

DuskTraum commented:
- Не надо в каждом тестовом методе добавлять еще проверку на наличие продукта, а потом создавать его если нет - это усложняет логику тестов. Если хочешь, чтобы тест запускался по порядку, то для тестовых методов можешь еще добавить Order(i).
- Не стоит комментировать каждый шаг в тесте, хороший тест сам показывает, что в нем делается

UlyanaRomanova committed:
WebDriverBasic. исправление

Вынесла авторизацию и переход на вкладку с продуктами в отдельные методы. Метод авторизации запускается в SetUp. Добавила порядок запуска тестов. Убрала Создание продукта при его отсутствии.

DuskTraum commented:
Принято
